src/rvsliblogger.cpp

```diff
@@ -347,15 +348,22 @@ int rvs::logger::JsonActionStartNodeCreate(const char* Module, const char* Actio
     rvs::logger::JsonStartNodeCreate(Module, Action);
   }
   isfirstrecord_m = true;
-  std::string row{RVSINDENT};
+  std::string row{newline};
+  if (isfirstaction_m){
+    isfirstaction_m = false;
+  } else{
+     std::cout << "not first ction"<<std::endl;
```

Comment:
"action" typo

Reply:
When will the fix be implemented? I just checked rvsliblogger.cpp and I can’t find the changes.